src/platform/common/application/applicationShell.ts

```diff
@@ -80,6 +81,7 @@ export class ApplicationShell implements IApplicationShell {
         ...items: T[]
     ): Thenable<T>;
     public showErrorMessage(message: any, options?: any, ...items: any[]) {
+        traceInfoIfCI(`Show Error Message ${message}, ${JSON.stringify(items)}`);
```

Comment:
Came across a flaky test where a dialog message appeared on CI, this is to see what was the last message that was displayed & caused tests to fail